osbs/conf.py

```diff
@@ -548,7 +548,7 @@ def get_platform_node_selector(self, platform):
         if platform:
             nodeselector_str = self._get_value("node_selector." + platform, self.conf_section,
                                                "node_selector." + platform)
-            if nodeselector_str:
+            if nodeselector_str and '=' in nodeselector_str:
```

Comment:
`node_selector.x86_64 = none` should set spec.nodeSelector to null.
`node_selector.x86_64 = amd64` should fail because 'amd64' is not a valid node selector.